test/test_jit_cuda_fuser.py

```diff
@@ -8,7 +8,7 @@
 from torch.testing._internal.codegen.random_topo_test import runDefaultTestWithSeed
 from torch.testing import FileCheck
 
-from test_jit import JitTestCase, RUN_CUDA
+from torch.testing._internal.jit_utils import JitTestCase, RUN_CUDA
```

Comment:
Note:
Importing from test_jit actually changes the behavior of eager mode for some reason. Will revisit this later.